topostats/default_config.yaml

```diff
@@ -42,8 +42,12 @@ grains:
   # Thresholding by area
   smallest_grain_size_nm2: 50 # Size in nm^2 of tiny grains/blobs (noise) to remove, must be > 0.0
   absolute_area_threshold:
-    above: [ 300, 3000 ] # above surface [Low, High] in nm^2 (also takes null)
-    below: [ null, null ] # below surface [Low, High] in nm^2 (also takes null)
+    above: [0, 10000000] # above surface [Low, High] in nm^2 (also takes null)
+    below: [null, null] # below surface [Low, High] in nm^2 (also takes null)
```

Comment:
The `300` to `3000` range was only really suiting `minicircle.spm` style images, and so in an effort to make the software more generally applicable by default, I've set it so that it won't remove any grains.

Reply:
In that case, would it make more sense to just set it to null?

Reply:
Good point, done 👍